pkg/rbeconfigsgen/rbeconfigsgen.go

```diff
@@ -496,12 +497,29 @@ func genCppConfigs(d *dockerRunner, o *Options, bazeliskPath string) (string, er
 	// 2. Harden each link.
 	// 3. Archive the contents of the config output directory into a tarball.
 	// 4. Copy the tarball from the container to the local temp directory.
-	out, err := d.execCmd("find", cppConfigDir, "-type", "l")
+	var out string
+	if o.ExecOS == "windows" {
+		out, err = d.execCmd("cmd", "/r", "dir", filepath.Clean(cppConfigDir), "/a:l", "/b")
+	} else {
+		out, err = d.execCmd("find", cppConfigDir, "-type", "l")
+	}
 	if err != nil {
-		return "", fmt.Errorf("unable to list symlinks in the C++ config generation build output directory: %w", err)
+		errMsg := fmt.Sprintf("unable to list symlinks in the C++ config generation build output directory: ")
+		// Windows `dir` has a non-zero exit status if no files are found.
+		// Linux just doesn't return any files but has a zero exit.
+		switch o.ExecOS {
+		case "windows":
+			out = ""
+			log.Printf("WARNING: %s%v", errMsg, err)
```

Comment:
Not a warning because it's not really actionable. Change to `Ignoring error indicating no symlinks were found in the Bazel output directory: %v`

No need to include errmsg because it would be confusing in the logs

Reply:
Done.